docs/user/dashboard/tsvb.asciidoc

```diff
@@ -1,126 +1,165 @@
 [[TSVB]]
 === TSVB
 
-*TSVB* requires a date field and supports <<aggregation-reference, most {es} metric aggregations>>, multiple visualization types, custom functions,
-and some math. 
-
-TIP: With *TSVB*, you can visualize the data from multiple series, but only *Timelion* can perform math across layers.
+*TSVB* enables you to visualize the data from multiple data series, supports <<aggregation-reference, 
+most {es} metric aggregations>>, multiple visualization types, custom functions, and some math. To use *TSVB*, you must have a date field. 
```

Comment:
you must have a date field > your data must have a date field?

Is it date field or time field?

Reply:
This is content that Wylie wrote, so I was hoping he would have commented. Since he is out this week, I'm going to leave it, but I'll make a note to follow up for the 7.13 release.